src/sympc/protocol/aby3/aby3.py

```diff
@@ -233,28 +235,43 @@ def local_decomposition(
             raise ValueError("ABY3 local_decomposition algorithm requires 3 parties")
 
         ring_size = int(ring_size)
+        ring_bits = get_nr_bits(session.ring_size)  # for bit-wise decomposition
```

Comment:
MAYBE: Can't we move this inside `if btiwise` (it is used only there)

Reply:
Modified